.github/actions/open-pr/action.yml

```diff
@@ -25,6 +25,9 @@ inputs:
   fail_if_files_unchanged:
     description: 'Fails the action if all of the specified files_to_commit are unchanged.'
     required: false
+  always_create_pr:
```

Comment:
Note that this action is facing a bit of feature creep but that's an existing problem. We can look into refactoring this action into 2 separate ones later on.

Reply:
/TODO: Refactor open-pr action